website/pages/docs/reference/cli/cloudquery.md

```diff
@@ -17,15 +17,15 @@ Find more information at:
 ### Options
 
 ```
-      --color string           Enable colorized output (on, off, auto) (default "auto")
-      --data-dir string        set persistent data directory (env: CQ_DATA_DIR) (default "./.cq")
-  -h, --help                   help for cloudquery
-      --log-console            enable console logging
-      --log-file-name string   Log filename (default "cloudquery.log")
-      --log-format string      Logging format (json, text) (default "text")
-      --log-level string       Logging level (default "info")
-      --no-log-file            Disable logging to file
-      --no-telemetry           disable telemetry collection
+      --color string             Enable colorized output (on, off, auto) (default "auto")
+      --data-dir string          set persistent data directory (env: CQ_DATA_DIR) (default "./.cq")
```

Comment:
Should we remove this too https://github.com/cloudquery/cloudquery/issues/2644 ?

Reply:
This is already handled here (This PR just needs a rebase) - https://github.com/cloudquery/cloudquery/pull/2796

Reply:
Rebased 👍